styleguide/js/zoolander.js

```diff
@@ -189,3 +180,17 @@ var url = window.location.pathname;
 if((url.match(/\/derek\/incubation\//gi) && url != '/derek/incubation/') || (url.match(/\/derek\/solutions\//gi) && url != '/derek/solutions/') || (url.match(/\/derek\/templates\//gi) && url != '/derek/templates/')){
     zoolanderSlideBtn.trigger('click');
 }
+
+
+//search
+var $searchContainer = $('#navbar-search');
+var $searchSubmit = $searchContainer.find('.navbar-icon-search');
+
+$searchSubmit.click(function() {
+  if ($searchContainer.hasClass('expanded')) {
+    $searchContainer.css({'width':'0'}).removeClass('expanded');
+  }
+  else {
+    $searchContainer.css({'width':'18%'}).addClass('expanded');
+  }
+});
```

Comment:
I'd think all of this functionality should live where the old functionality lived (i.e. instead the dom ready), and not just sitting here separately.